ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1081,15 +1081,15 @@ IE_CPU.acosh
 
 # Interpolate-1 in linear mode
 # 2.666666507720947266 is not close to 3 at index 1
-IE_CPU.interpolate_down_scales_const_linear
+# IE_CPU.interpolate_down_scales_const_linear
 
 # Result mismatch 2.666666507720947266 is not close to 3 at index 1
-IE_CPU.onnx_resize10_down_scales_const_linear
-IE_CPU.onnx_resize11_scales_down_linear
+# IE_CPU.onnx_resize10_down_scales_const_linear
+# IE_CPU.onnx_resize11_scales_down_linear
 
 # C++ exception with description "Interpolate operation should be converted to Interp
 # not constant output_shape passed to Interpolate-1
-IE_CPU.onnx_resize11_scales_nearest_asymmetric_floor_dynamic_sizes
+# IE_CPU.onnx_resize11_scales_nearest_asymmetric_floor_dynamic_sizes
```

Comment:
Please remove commented test cases if you enable them

Reply:
This test was disabled, because current IE CPU implementation of `Interpolate` satisfies the specification of `Interpolate-1`, not `Interpolate-4`.